src/attpc_engine/kinematics/angle.py

```diff
@@ -0,0 +1,80 @@
+from typing import Protocol
+from numpy.random import Generator
+import numpy as np
+
+
+class PolarDistribution(Protocol):
+    """Protocol for defining a reaction polar angle distribution in the CM frame
+
+    Note that for polar angle distributions, the coordinate to sample
+    is cos(polar) not polar (to cover solid angle uniformly). This means
+    hat the valid domain of these distributions is [0.0, pi]
```

Comment:
I think this should be "that the valid..."

Reply:
Fixed